lib/database_connection.rb

```diff
@@ -0,0 +1,15 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  class << self
```

Comment:
never seen this - is it effectively a self.attr_reader? (i know that's not a thing)